wkhtmltopdf-image/src/main/docker/app.py

```diff
@@ -0,0 +1,138 @@
+#! /usr/bin/env python
+import StringIO
+import json
+import logging
+import uuid
+from os import EX_OK
+from subprocess import Popen, PIPE
+
+from werkzeug.exceptions import HTTPException, NotFound
+from werkzeug.routing import Map, Rule
+from werkzeug.wrappers import Request, Response
+
+"""
+This python server is responsible for generating PDFs from HTML
+using an underlying wkhtmltopdf command-line tool.
+
+To use this application, the user must send a POST request with
+base64 or form encoded encoded HTML content and the wkhtmltopdf Options in
+request data, with keys 'contents' and 'options'.
+The application will return a response with the PDF file.
+"""
+class WkHtmlToPdfServer():
+    def __init__(self):
+        self.url_map = Map([
+            Rule('/', endpoint='generate_pdf'),
+            Rule('/health', endpoint='health_check')
+        ])
+        logging.basicConfig()
+        self.logger = logging.getLogger('WkHtmlToPdfServer')
+        #TODO allow environment to configure this
+        self.logger.setLevel(logging.INFO)
```

Comment:
Added some minor logging.  We need to look into Best Practices for configuration logging for a python application